corehq/apps/importer/util.py

```diff
@@ -95,6 +97,25 @@ def from_request(cls, request):
 
         )
 
+
+def open_workbook(filename=None, **kwargs):
+    _soft_assert = soft_assert(notify_admins=True)
+    try:
+        return xlrd.open_workbook(filename=filename, **kwargs)
+    except xlrd.XLRDError as e:
+        message = unicode(e)
+        if message == u'Workbook is encrypted':
+            raise ImporterExcelFileEncrypted(message)
+        else:
+            _soft_assert(False, 'displaying XLRDError directly to user', e)
+            raise ImporterExcelError(message)
+    except IOError as e:
+        raise ImporterFileNotFound(unicode(e))
+    except Exception as e:
+        _soft_assert(False, 'xlrd.open_workbook raised unaccounted for error', e)
+        raise ImporterFileNotFound(unicode(e))
```

Comment:
Should this `raise ImporterError(...)` since it is a catchall, not necessarily "file not found"? Also, consider raising the original exception as recommended above to preserve the stack trace.